mozilla/fx-private-relay
# Holiday promo banner ga ping and UTM params

# New feature description

Added a GA ping to the "Get 1 year of premium button" and UTM params to the link

# Screenshot (if applicable)

![image](https://github.com/mozilla/fx-private-relay/assets/59676643/46ae721c-475a-4792-b456-cf840576cad5)

# How to test
(Make sure holiday_promo_2023 flag is enabled)
1. Go to the home page and premium page as a non-signed in user.
2. Click the "Get 1 year of premium" button.
3. Verify in your dev tools console that the GA ping was sent.


# Checklist (Definition of Done)
